adminpages/member-edit/pmpro-class-member-edit-panel-subscriptions.php

```diff
@@ -200,6 +201,26 @@ private function display_subscription_table( $subscriptions ) {
 							// Show the subscription fee.
 							echo esc_html( $subscription->get_cost_text() );
 							?>
+						</td>
+						<td>
+							<?php
+							// Show the gateway used for this subscription.
+							$subscription_gateway = $subscription->get_gateway_object();
+							if ( ! empty( $subscription_gateway->gateway ) ) {
+								$gateway = esc_html( ucwords( $subscription_gateway->gateway ) );
+
+								// Check if the gateway was set to sandbox mode.
+								if ( $subscription_gateway->gateway_environment === 'sandbox' ) {
```

Comment:
Getting an issue here in my dev:

Warning: Undefined property: PMProGateway_paypalexpress::$gateway_environment in /Users/kimberlycoleman/Local Sites/pmprokimfork/app/public/wp-content/plugins/paid-memberships-pro/adminpages/member-edit/pmpro-class-member-edit-panel-subscriptions.php on line 213